src/styles/index.scss

```diff
@@ -1,3 +1,48 @@
+@import './variables';
+@import './card';
+@import './stars';
+@import './header';
+
+@font-face {
+  font-family: Roboto;
+  src: url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
```

Comment:
The @font-face rule is importing the Roboto font from Google Fonts. Since you're already linking to Google Fonts in your HTML, this @font-face rule might be redundant. Consider removing it to avoid unnecessary requests.